sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1984,6 +2022,20 @@ void CAFMaker::produce(art::Event& evt) noexcept {
   // rec.hdr.blind = 0;
   // rec.hdr.filt = rb::IsFiltered(evt, slices, sliceID);
 
+  // Fill the header info for the given event's spill quality info
+  if ( fHasBNBInfo && fHasNuMIInfo ) {
+    std::cout << "Found > 0 BNBInfo size and NuMIInfo size, which seems strange. Will not fill event-specific spill quality info for this event..." << std::endl;
+  }
+  else if ( fHasBNBInfo ) {
+    if ( fBNBInfoEventMap.find(evt.id().event()) == fBNBInfoEventMap.end() )
```

Comment:
Or, more intelligible:
```suggestion
    unsigned int const eventNo = evt.id().event();
    if ( fBNBInfoEventMap.count(eventNo) == 0 )
```

Reply:
Wait shouldn't count > 0 if `fHasBNBInfo` ?

Reply:
That count "counts" the number of entries in the map _for `eventNo`_: there may be some information (in which case presumably `fHasBNBInfo` is `true`), but the one for `eventNo` might be missing, in which case that `count(eventNo)` returns `0`.
I would call that situation a problem, but technically it is possible.